cookbook/map.rst.inc

```diff
@@ -41,6 +41,7 @@
 * :doc:`/cookbook/console/index`
 
   * :doc:`/cookbook/console/console_command`
+  * :doc:`/cookbook/console/command_in_controller`
```

Comment:
Given the order in the index file, you should move this below the next line for consistency.